src/preamble.js

```diff
@@ -1501,5 +1594,48 @@ function lookupSymbol(ptr) { // for a pointer, print out all symbols that resolv
 
 var memoryInitializer = null;
 
-// === Body ===
+#if USE_PTHREADS
```

Comment:
I don't expect that the piece of code in this location would grow much, since it's just to avoid the lack of support for `navigator.hardwareConcurrency` in the browser.